strictEqual.js

```diff
@@ -0,0 +1,9 @@
+const strictEquals = (valueA, valueB) => {
+  if (Object.is(valueA, valueB)) {
+    return true
+  } else {
+    return false
+  }
+}
+
```

Comment:
Object.is() returns true or false so you can just return Object.is() 
`const strickEquals = (valuseA, valueB) => { return (Object.is(valueA, valueB) }`